### ISSUE TITLE
Wrong bundle id taken from target

### ISSUE THREAD

**Issue by lukaswuerzburger** [lyndsey-ferguson/get_product_bundle_id#2]
### Problem

I have different bundle IDs for one target with different build configurations:
I have four build configurations: `Debug`, `Debug Stage`, `Release`, `Release Stage`. In my project file it is exactly in this order:
```
//:configuration = Debug Stage
PRODUCT_BUNDLE_IDENTIFIER = com.myproj.mobile-stage

//:configuration = Release
PRODUCT_BUNDLE_IDENTIFIER = com.myproj.mobile

//:configuration = Debug
PRODUCT_BUNDLE_IDENTIFIER = com.myproj.mobile

//:configuration = Release Stage
PRODUCT_BUNDLE_IDENTIFIER = com.myproj.mobile-stage
```
So, when I build the stage app it works fine. But when I build the production app with the `Debug` or `Release` configuration it still chooses `com.myproj.mobile-stage` because `get_product_bundle_id` always **takes the first** configuration.

### Solution Suggestion

Add a new parameter to specify the build configuration

**Comment by lyndsey-ferguson** [comment]
@lukaswuerzburger thanks for the report. To make it super easy for me, can you provide a sample project and a sample `Fastfile`? 

I don't have a lot of time to work on my open source right now so any removal of obstacles or speed bumps would help me immensely.

**Comment by ozgur** [comment]
Any update on getting this fixed?

**Comment by lukaswuerzburger** [comment]
Hey @ozgur. Thanks for reminding. I'll provide a test project today so that @lyndsey-ferguson can have a look. I'll also create a PR.

**Comment by lukaswuerzburger** [comment]
I'm having trouble installing the latest version on my computer. I specified `gem 'fastlane-plugin-get_product_bundle_id', '0.1.3'` in the `PluginFile`. But it is still falling back to 0.1.2. So I'm writing it down here:

<h3>Steps to reproduce</h3>

In my case I want to use one target but multiple configurations for Staging and Production:

**1. Clone "Debug" and call it "Debug Staging". Clone "Release" and call it "Release Staging"**

![Screen Shot 2019-11-26 at 12 43 19 PM](https://user-images.githubusercontent.com/10812458/69627468-ee4efe00-104a-11ea-92a5-5a1c68f235ff.png)

**2. Change the bundle identifiers for the staging app**

![Screen Shot 2019-11-26 at 12 43 40 PM](https://user-images.githubusercontent.com/10812458/69627584-366e2080-104b-11ea-8ec4-9bc3c241eabf.png)

**3. Create a new scheme for staging app**

![Screen Shot 2019-11-26 at 12 45 41 PM](https://user-images.githubusercontent.com/10812458/69627663-5d2c5700-104b-11ea-98d7-632448904dab.png)

**4. Change the scheme's build configuration**

![Screen Shot 2019-11-26 at 12 45 58 PM](https://user-images.githubusercontent.com/10812458/69627717-71705400-104b-11ea-9dbc-54bc6af7aa22.png)

**5. Run `get_product_bundle_id`**

- project_filepath: "./MultiConfiguration.xcodeproj"
- scheme: "MultiConfigurationStage"

**Expected output:** com.company.MultiConfigurationStage
**Actual output:** com.company.MultiConfiguration

It returns whatever is first in the list of configurations. As you can see in the screenshots 1,2 and 4 "Debug" is the first configuration in the list and points to "com.company.MultiConfiguration". I've found the lines of code as well: 
https://github.com/lyndsey-ferguson/get_product_bundle_id/blob/c5edf20002b9cd2b78316676651e4cb2cce96b76/lib/fastlane/plugin/get_product_bundle_id/actions/get_product_bundle_id_action.rb#L14
https://github.com/lyndsey-ferguson/get_product_bundle_id/blob/c5edf20002b9cd2b78316676651e4cb2cce96b76/lib/fastlane/plugin/get_product_bundle_id/actions/get_product_bundle_id_action.rb#L20

**Comment by lyndsey-ferguson** [comment]
Fixed in [v0.1.4](https://github.com/lyndsey-ferguson/get_product_bundle_id/releases/tag/v0.1.4)

**Comment by lyndsey-ferguson** [comment]
@lukaswuerzburger Thanks for the feature!
